common/djangoapps/student/tests/test_views.py

```diff
@@ -0,0 +1,56 @@
+"""
+test views
+"""
+import ddt
+from mock import patch
+from django.core.urlresolvers import reverse
+from certificates.models import CertificateStatuses
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+@ddt.ddt
+class TestStudentDashboardUnenrollments(ModuleStoreTestCase):
+    """
+    Test to ensure that the student dashboard does not show the unenroll button for users with certificates.
+    """
+    USERNAME = "Bob"
+    EMAIL = "bob@example.com"
+    PASSWORD = "edx"
+    UNENROLL_ELEMENT = '<li class="actions-item" id="actions-item-unenroll-0">'
```

Comment:
That is indeed 3-clause BSD, which is fine. Also, since you're not copying code into our project, but only referencing the project in a requirements file, it's fine either way.